components/Dropdown/Dropdown.js

```diff
@@ -1,28 +1,33 @@
 class Dropdown {
   constructor(element) {
-    
     // Assign this.element to the dropdown element
-    this.element;
-    
+    this.element = element;
+
     // Get the element with the ".dropdown-button" class found in the dropdown element (look at the HTML for context)
-    this.button = this.element.querySelector();
-    
+    this.button = this.element.querySelector(".dropdown-button");
+
     // assign the reference to the ".dropdown-content" class found in the dropdown element
-    this.content;
-    
-    // Add a click handler to the button reference and call the toggleContent method.
-    this.button.addEventListener('click', () => {
+    this.content = this.element.querySelector(".dropdown-content");
+    this.show = false; // GSAP
 
-    })
+    // Add a click handler to the button reference and call the toggleContent method.
+    this.button.addEventListener("click", () => this.toggleContent());
   }
 
   toggleContent() {
-    
-    // Toggle the ".dropdown-hidden" class off and on
-    this.content;
+    // GSAP
+    if (this.show === false) {
+      this.show = true;
+      TweenMax.to(this.content, 0, { display: "flex" });
+      TweenMax.to(this.content, 2, { opacity: 1 });
+    } else {
+      this.show = false;
+      TweenMax.to(this.content, 0.5, { opacity: 0, display: "none" });
+    }
```

Comment:
excellent use of GSAP here to get a nice, smooth fade effect! remember in your conditional since it checks if a statement or `variable` renders as false or true, you could say `if (this.show)` to check if it's true, or `if (!this.show)` if you want the conditional to check if it's false. not as semantic but it will be a lot more prevalent in the future and will save you a little time